RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: minor
```

Comment:
can we change this to patch? :)

Reply:
Was just following the recommendations in SEMVER:
> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.

Considering it seemed like I was adding functionality yet maintaining backwards compatibility and it wasn't a bugfix, I figured it was a minor release. But if you feel it should be a patch I'd be happy to change it.

Reply:
I'd say we are not following semver until we release version 1.0.0 :D

But that's ok, no need for a change ;)